src/reporters/lang/en.js

```diff
@@ -188,6 +188,8 @@ const messages = {
 
   execMissingCommand: 'Missing command name.',
 
+  dashDashDeprecation:
+    "Using -- to pass arguments to your scripts isn't required anymore. Doing this may cause issues in future versions.",
```

Comment:
Right, in that case then, maybe this: "From Yarn 1.0, scripts do not require \`--\` to be passed. In a future version of Yarn, passing \`--\` will be forwarded to the script directly." ?